docs/placeholders.md

```diff
@@ -0,0 +1,109 @@
+# Placeholders
+DiscordSRV contains many placeholders which allows you to customise messages entirely to your liking.
+
+Information about which placeholders can be used where can be found in the config. 
+
+---
+# Formatting
+
+!!! note "General Formatting Help"
+    Information on formatting Minecraft messages can be found [here](https://github.com/Vankka/EnhancedLegacyText/wiki/Format). 
+    
+    Information on Discord markdown can be found [here](https://support.discord.com/hc/en-us/articles/210298617)
+
+## Checking for empty placeholders
+You can take empty placeholders into account and use an alternate placeholder instead.
+
+For example `%player_meta_prefix|player_prefix%`. This first looks for `%player_meta_prefix%` if that is empty, than `%player_prefix%` is used instead.
+
+## Placeholder Suffixes
+Placeholders which provide a list of items can have a suffix set between each item to separate them.
+
+For example `%user_roles:', '%`, lists the users roles seperated by a comma and a space (`', '`).
+
+## Relookup Placeholders
+Relookup placeholders are a special type of placeholder which allow you to specify what information you want from it.
```

Comment:
Dunno if this is the term we wanna use, I'm open to ideas. Relookup sounds slightly misleading / technical in my opinion

Reply:
Changed to recursive as discussed over Discord